server/src/helpers/tjdb_dto_transforms.ts

```diff
@@ -32,7 +32,7 @@ const transformationsByVersion = {
     });
     dto.schema = {
       columns: transformedColumns,
-      foreign_keys: [],
+      foreign_keys: dto.schema.foreign_keys || [],
```

Comment:
For versions which was released before 2.42.0 will not have foreign_keys right?
Why do we have to add it? 🤔